corehq/apps/accounting/forms.py

```diff
@@ -248,35 +248,29 @@ def update_basic_info(self, account):
         contact_info.save()
 
 
-class BillingAccountContactForm(forms.Form):
-    first_name = forms.CharField(label='First Name', required=False)
-    last_name = forms.CharField(label='Last Name', required=False)
-    company_name = forms.CharField(label='Company Name', required=False)
-    phone_number = forms.CharField(label='Phone Number', required=False)
-    address_line_1 = forms.CharField(label='Address Line 1')
-    address_line_2 = forms.CharField(label='Address Line 2', required=False)
-    city = forms.CharField()
-    region = forms.CharField(label="State/Province/Region")
-    postal_code = forms.CharField(label="Postal Code")
```

Comment:
Note that removing this block causes the form to use the "verbose name" attached to the model, which seems just as sufficient unless we're really picky about the names.